hack/lib/util/misc.sh

```diff
@@ -175,19 +175,17 @@ function os::util::curl_etcd() {
 		local etcd_client_cert_p12_password="${CURL_CERT_P12_PASSWORD:-'password'}"
 		if [[ ! -f "${etcd_client_cert_p12}" ]]; then
 			openssl pkcs12 -export                        \
-			               -in "${etcd_client_cert}"      \
-			               -inkey "${etcd_client_key}"    \
+			               -in "${etcd_client_cert[1]}"   \
+			               -inkey "${etcd_client_key[1]}" \
 			               -out "${etcd_client_cert_p12}" \
 			               -password "pass:${etcd_client_cert_p12_password}"
 		fi
 
-		curl --fail --silent --cacert "${ca_bundle}" \
-		     --cert "${etcd_client_cert_p12}:${etcd_client_cert_p12_password}" "${full_url}"
+		etcd_client_cert=(--cert "${etcd_client_cert_p12}:${etcd_client_cert_p12_password}")
+		unset etcd_client_key
 	fi
```

Comment:
why not just return here or put the other case in an `else`? the rewrite is weird